defender/secret-example.yml

```diff
@@ -2,11 +2,59 @@ keys:
   api: <API Key goes here>
   secret: <API Secret goes here>
 
+# WARNING: When deploying, secrets listed below will be overwrite the ones in Defender.
+# If secrets do not need to be updated, comment them out in from your serverless.yml files.
+# Note: The overwrite is limited to the stack that you are deploying.
 secrets:
-  # Twitter Keys
-  # Consumer Keys from an elevated developer account
-  appKey: ""
-  appSecret: ""
-  # Authentication Tokens (must have write permissions)
-  accessToken: ""
-  accessSecret: ""
+  ctoken_monitor:
+    discordWebhook: ""
+  datadog_alerts_heat_map:
+    datadogApiKey: ""
+  datadog_alerts_heat_map_dev:
+    datadogApiKey: ""
+  datadog_forta_bot_alerts:
+    datadogApiKey: ""
+  datadog_forta_bot_alerts_dev:
+    datadogApiKey: ""
+  datadog_forta_detection_bot_health:
+    datadogApiKey: ""
+  forta_ctoken:
+    discordWebhook: ""
+  forta_distribution:
+    discordWebhook: ""
+  forta_governance:
+    discordWebhook: ""
+  forta_large_borrows_governance:
+    discordWebhook: ""
+  forta_large_delegations:
+    discordWebhook: ""
+  forta_low_liquidity:
+    discordWebhook: ""
+  forta_multi_sig:
+    discordWebhook: ""
+  forta_oracle_price:
+    discordWebhook: ""
+  forta_underlying_asset:
+    discordWebhook: ""
+  forta_v2_liquidation_monitor_dev:
+    discordWebhook: ""
+  forta_v3_liquidation_monitor_dev:
+    discordWebhook: ""
```

Comment:
we can also add the `gasless_voting_dev` and `governance_automation_dev` as well

Reply:
Same as above. Some of the autotasks are 100% self-contained and do not require keys, URLs, webhooks addresses, etc. to be shared with them via Defender secrets. So secrets do not need to be defined for these stacks.